--- /var/lib/sbopkg/SBo-git/network/slurm/slurm.SlackBuild	2024-05-06 19:33:37.491059175 -0500
+++ /var/lib/sbopkg/SBo-git/network/slurm/slurm.SlackBuild.new	2024-06-07 10:01:31.026284135 -0500
@@ -82,6 +82,10 @@
 RRDTOOL_SUPPORT=""
 [ "$RRDTOOL" = "yes" ] && RRDTOOL_SUPPORT="--with-rrdtool"
 
+PMIX=${PMIX:-no}
+PMIX_SUPPORT=""
+[ "$PMIX" = "yes" ] && PMIX_SUPPORT="--with-pmix=/usr/local"
+
 rm -rf $PKG
 mkdir -p $TMP $PKG $OUTPUT
 cd $TMP
@@ -108,6 +112,7 @@
   $DEPRECATED \
   $HWLOC_SUPPORT \
   $RRDTOOL_SUPPORT \
+  $PMIX_SUPPORT \
   --build=$ARCH-slackware-linux \
   --host=$ARCH-slackware-linux
 
